database/migrations/2017_03_31_211222_create_feedback_tables.php

```diff
@@ -0,0 +1,352 @@
+<?php
+
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFeedbackTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
In response to your previous question, this will need to include the new permissions (all permissions are currently migration-controlled)